metaflow/decorators.py

```diff
@@ -239,6 +239,13 @@ def package_init(self, flow, step_name, environment):
         """
         pass
 
+    def add_to_package(self):
+        """
+        Called to add custom packages needed for a decorator.
+        Returns a list of tuples where each tuple represents (file_path,arcname)
```

Comment:
please detail what `file_path` and `arcname` here represent

Reply:
The `MetaflowPackage` class consists of a `path_tuples` function. This instance method helps seek the files to include for the code package. The `path_tuples` compiles the tarball of all the code files to include for execution. The function returns a list of tuples which consists of `file_path` and `arcname`; The `filepath` is the path of the file in the local FS to include and the `arcname` is the name/path of the file in the tarball when it is unpacked.

Reply:
👍  thanks!

Reply:
Ditto